imputation/tsImpute_test.go

```diff
@@ -0,0 +1,173 @@
+package imputation
+
+//TODO(eftekhari-mhs): Refactor to table-driven tests: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests
+
+import (
+    "testing"
+    "errors"
+)
+
+func computeMean(s TimeSeries) (float64, error){
```

Comment:
In this particular function designed only for testing, it might suffice to not check for division by zero and just dying if given an empty time series.  This is because the test writer should know to not call the function on such a time series.  If e does, the test will crash or fail.  In regular user code where one has to handle all cases gracefully.

Reply:
Is this because developer does the testing and not regular user?